delta-app/src/main/java/io/cdap/delta/app/DeltaWorker.java

```diff
@@ -186,8 +188,8 @@ public void initialize(WorkerContext context) throws Exception {
     readerDefinition = new EventReaderDefinition(expandedTables,
                                                  config.getDmlBlacklist(),
                                                  config.getDdlBlacklist());
-
-    eventQueue = new ArrayBlockingQueue<>(100);
+    eventQueueSize = Integer.parseInt(context.getRuntimeArguments().getOrDefault(EVENT_QUEUE_SIZE, "10"));
```

Comment:
should we set the default queue size to 100?

Reply:
Before merge, can we change the default value to 100, feel like 10 is too small. 
Also, is there any doc for delta app. Maybe it is good to show how to use the runtime argument to set the customized blocking queue size.

Reply:
I'll add more comments, but if we get 100 update events, that means we have roughly 2gb / 100 = 20mb per event, which means roughly 10mb per row since updates contain the data from both before and after. So an OOM is within the realm of possibility if there are binary columns.

I think it's better to set defaults that skew towards reliability, and let the more advanced users adjust for performance. I've also opened https://issues.cask.co/browse/CDAP-16755 to do this based on event size rather than number of events.